pkg/processor/trigger/kafka/test/kafka_test.go

```diff
@@ -272,13 +277,21 @@ func (suite *testSuite) TestEventRecorderRebalance() {
 			}
 
 			// make sure they are all read
-			time.Sleep(3 * time.Second)
+			var receivedBodies1, receivedBodies2 []string
+			err := common.RetryUntilSuccessful(15*time.Second,
+				2*time.Second,
+				func() bool {
+					receivedBodies1 = suite.resolveReceivedEventBodies(deployResult)
+					receivedBodies2 = suite.resolveReceivedEventBodies(newDeployResult)
+
+					// make sure that new events were received in both functions
+					return len(receivedBodies1) > len(receivedBodies) && len(receivedBodies2) >= int(suite.NumPartitions)/2
+
+				})
+			suite.Require().NoError(err, "Failed to get events")
```

Comment:
+ get container logs before bailing
+ imo, until resolved, comment out the "rebalance" test and fix it on a side. Ive ran this test multiple time from both my PC and  MAC and it looks fine to me.